test/src/main/java/io/varhttp/filterorder/FilterControllerClass.java

```diff
@@ -0,0 +1,29 @@
+package io.varhttp.filterorder;
+
+import io.varhttp.Controller;
+import io.varhttp.ControllerClass;
+import io.varhttp.HttpMethod;
+import io.varhttp.controllers.withfilters.FilterCatcher;
+
+import javax.inject.Inject;
+
+@ClassFilter1(20)
```

Comment:
Does this number set the order that they should be called?

Reply:
should be fine now :-)